### PR TITLE
docs: changes throughout for phrasing and additional detail.

### PR DESCRIPTION
**Pull request by jonxstill**
## Change Summary

Updates to Niels' AVD Beginner TOI for phrasing/language and a little extra detail in various places.

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.<role-name>`

## Proposed changes
Changes for language/phrasing as requested.

### User Checklist

- N/A

### Repository Checklist

- [ ] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 1]
```diff
@@ -266,7 +299,7 @@ l3leaf:
     spines: [DC1-SPINE1, DC1-SPINE2]
     uplink_to_spine_interfaces: [Ethernet1, Ethernet2]
     mlag_interfaces: [Ethernet3, Ethernet4]
-    spanning_tree_priority: 4096
+    spanning_tree_priority: 409
```
Comment (nielsjlarsen) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md:302]: Can you elaborate? Is this just a typo?

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md [hunk 2]
```diff
@@ -1,115 +1,144 @@
...
+## What are the requirements to run Ansible?
+
+Ansible can run on almost anything, but in production scenarios Ansible is typically deployed on a virtual Linux server,
 running on the customer’s preferred hypervisor. This Ansible server then communicates either directly with the
-Arista network devices or with Arista CloudVision Portal, which in turn communicates with the Arista network devices.
-Controlling what Ansible does is typically done using a SSH terminal session to the Ansible server from the Operator’s computer.
-Using Ansible to control an Arista network infrastructure is agentless,
-i.e. it requires nothing additional to be installed on the Arista network devices or on Arista CloudVision Portal and only uses the existing APIs to remotely configure the network.
+Arista network devices via eAPI or with Arista CloudVision Portal, which in turn communicates with the Arista network devices.
+Controlling what Ansible does is typically done using an SSH terminal session to the Ansible server from the Operator’s computer.
```
Comment (nielsjlarsen) [inline ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md:83]: Shouldn't "using an SSH terminal session" be "using **a** SSH terminal session"?

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md (hunk 1 above)
Comment (jonxstill) [reply ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md:302]: Yes, just a typo - I must have accidentally deleted a char.

File: ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md (hunk 2 above)
Comment (jonxstill) [reply ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md:83]: "a SSH" sounds wrong to me.  "an SSH" sounds correct.
This is the best explanation I can find:
https://www.grammar.com/a-vs-an-when-to-use

In this case the S at the beginning of SSH sounds like "ehss" - that 'eh' sound means 'an' should be used.

**Review by nielsjlarsen — APPROVED** [review]

**Pull request merged**